LICENCES-3RD-PARTY.txt

```diff
@@ -0,0 +1,15 @@
+Ckan_client-PHP
+---------------
+
+Copyright (c) 2010 Jeffrey Barke http://jeffreybarke.net/ https://github.com/jeffreybarke/Ckan_client-PHP Licensed under the MIT license Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+PHP Markdown
+------------
+
+Ckan_client.php includes a copy of Michel Fortin's PHP Markdown (copyright (c) 2004-2009 Michel Fortin http://michelf.com/. All rights reserved.) which is based on on Markdown (copyright (c) 2003-2006 John Gruber http://daringfireball.net/. All rights reserved.).
+
+Read Me: ./lib/php_markdown/PHP Markdown Readme.text License: ./lib/php_markdown/License.text
```

Comment:
Would be clearer if it refered to the full path, so `helpers/CKAN_Code/lib/php_markdown/License.text` and `helpers/CKAN_Code/lib/php_markdown/PHP%20Markdown%20Readme.text`